app/views/teaching_events/show/_how-to-attend.html.erb

```diff
@@ -1,12 +1,36 @@
-<section class="teaching-event__how-to-attend">
-  <h2>How to attend</h2>
+<%
+    if @event.open? && [
```

Comment:
Is it worth replacing the `allow_registration?` method with this logic?

Reply:
Yeah, calling helpers from the presenter didn't feel quite right and I didn't want to duplicate them. Having it in a single method would be much nicer.